### DISCUSSION TITLE
the dynamic type `bevy_reflect::DynamicStruct` (representing `glam::IVec2`) does not support hashing

### DISCUSSION THREAD

**Discussion by Liam19** [bevyengine/bevy#16656] [Q&A]
As the title suggests, I'm getting this error when updating my project to Bevy 0.15.0. This error occurs when trying to use `DynamicScene::write_to_world()`. I can't find/understand any of the reflection migration guides for 0.15.0, so am I doing something wrong? This was working in 0.14.2.

**Comment by jonathandw743** [comment]
use a HashMap<IVec2, ()> instead of a HashSet<IVec2>
